### PR TITLE
block_writer: use pgwire proto

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

**Comment by petermattis** [comment]
LGTM. It would be great to fix the `bank/sql_bank` and `fakerealtime` examples as well. Pretty please.

**Comment by dt** [comment]
Updated this one to fix docs as well. PTAL again, sorry.

File: block_writer/README.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ Run against an existing cockroach node or cluster.
 ```
 # Build cockroach binary from https://github.com/cockroachdb/cockroach
 # Start it in dev mode (listens on localhost:26257)
-./cockroach start --dev
+./cockroach start --insecure
```
Comment (petermattis) [inline block_writer/README.md]: Perhaps this section should be taken out as there is no difference between Development and Insecure mode now.

**Comment by petermattis** [comment]
Still looks good.

**Comment by tamird** [comment]
Does this close #29?

**Comment by dt** [comment]
Oh, whoops. Yes, I think it does.

File: block_writer/README.md [hunk 2]
```diff
@@ -34,5 +23,5 @@ Run against an existing cockroach node or cluster.
 # Launch your node or cluster in secure mode with certificates in [mycertsdir]
 # Find a reachable address:[mycockroach:26257].
 # Run the example with:
-./block_writer https://mycockroach:26257/?certs=mycertsdir
+./block_writer "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
```
Comment (tamird) [inline block_writer/README.md]: why the quotes?

Comment (dt) [inline block_writer/README.md]: `&` in the URL.

Comment (tamird) [inline block_writer/README.md]: as mentioned in #39, single quotes clarify that there's no expansion

**Pull request merged**